wandb/sdk/wandb_artifacts.py

```diff
@@ -1805,15 +1790,185 @@ def _content_addressed_path(md5: str) -> FilePathStr:
         )
 
 
+class AzureHandler(StorageHandler):
+    def can_handle(self, parsed_url: "ParseResult") -> bool:
+        return parsed_url.scheme == "https" and parsed_url.netloc.endswith(
```

Comment:
Way to go MS. I really want to know who/why they decided to just use https instead of their own uri scheme 🤨

Reply:
Most resources on the internet don't use a custom scheme / protocol. You can configure the storage account to be publicly accessible, and then supporting the HTTP scheme lets people access the blobs with a web browser.